### PR TITLE
Add tenant support for all other emulator endpoints

### PR DESCRIPTION
**Pull request by lisajian**
### Description

Added the following endpoint to support multi-tenancy op's:

- `/emulator/v1/projects/{targetProjectId}/tenants/{tenantId}/accounts`

Regenerated `apiSpec.js` and `schema.ts` with this update

Corresponding internal bug: b/192387245

### Scenarios Tested

`npm run generate:auth-api`
`npm test` passes

### Sample Commands

N/A

### CODE REVIEW

File: src/test/emulators/auth/misc.spec.ts [hunk 1]
```diff
@@ -527,4 +581,18 @@ describeAuthEmulator("emulator utility APIs", ({ authApi }) => {
           .equals("Users are present, unable to set passthrough mode");
       });
   });
+
+  it("should error on updating usageMode on PATCH /emulator/v1/projects/{PROJECT_ID}/tenants/{TENANT_ID}/config", async () => {
```
Comment (yuchenshi) [inline src/test/emulators/auth/misc.spec.ts:585]: This whole API makes no sense. As you've observed, `usageMode` cannot be set per tenant, and I don't think `allowDuplicateEmails` can be set per tenant in production either. Shouldn't tenant inherit these two settings from agent project config?

File: scripts/gen-auth-api-spec.ts [hunk 2]
```diff
@@ -399,6 +438,73 @@ function addEmulatorOperations(openapi3: any): void {
       tags: ["emulator"],
     },
   };
+  openapi3.paths["/emulator/v1/projects/{targetProjectId}/tenants/{tenantId}/config"] = {
```
Comment (yuchenshi) [inline scripts/gen-auth-api-spec.ts:441]: see comment above

File: src/test/emulators/auth/misc.spec.ts (hunk 1 above)
Comment (lisajian) [reply src/test/emulators/auth/misc.spec.ts]: Yeah, that all makes sense, thanks for the clarification - I'll go ahead and revert the APIs and if they're needed later on, we can address it then 👍👍

File: scripts/gen-auth-api-spec.ts (hunk 2 above)
Comment (lisajian) [reply scripts/gen-auth-api-spec.ts]: Removed

**Review by yuchenshi — APPROVED** [review]

**Pull request merged**